cookbook/profiler/data_collector.rst

```diff
@@ -131,15 +131,15 @@ following example can help you get started:
         </div>
         {% endset %}
 
-        {# Set the "link" value to false if you do not have a big "panel" 
+        {# Set the "link" value to false if you do not have a big "panel"
            section that you want to direct the user to. #}
-        {% include '@WebProfiler/Profiler/toolbar_item.html.twig' with { 'link': true } %}
+        {{ include('@WebProfiler/Profiler/toolbar_item.html.twig', variables = { 'link': true }) }}
```

Comment:
Good catch, Javier. I don't think we should talk about named arguments here at all as it's a too common concept in Twig to be mentioned in every Twig function we use.